docs/tutorial/exporting_targets.rst

```diff
@@ -36,6 +36,10 @@ option to ``ON`` will mark all active targets in the above list as ``EXPORTABLE`
     marked ``EXPORTABLE`` must be added to the same export set.  Failure to add them will
     result in a CMake error in the exporting project.
 
+.. note:: The recommended usage of the HIP targets is via the ``blt::hip`` and
+    ``blt::hip_runtime`` aliases. Alias targets cannot be exported, so the
+    ``blt_hip``/``blt_hip_runtime`` names can be used for this purpose.
+
```

Comment:
Should we mention anything about the underlying targets, and if those are still available for users?
(I think those are `hip::host` and `hip::device`, but I'm not sure)

Reply:
They are available, but `hip::host` actually seems broken (missing include) so I don't think we should recommend it.

Reply:
The behavior would also be different would it not?

I admit I'm wondering, especially with this doc change, if having the aliases is worth it since now there's a split between at least two uses.  Should it just always be `blt_<backend>` perhaps rather than have both?  At least that way it can be made consistent everywhere. Really not fond of how aliases only kinda-sorta work across projects like this.

Reply:
😭 

Installing a target provided by BLT is a pretty specific use case that afaik only a couple of projects are doing. Using `blt_<backend>` is fine but just looks wrong to me, I feel like the namespaced target is correct since we want people to think of them as imported targets.